### ISSUE TITLE
Multiple characters not showing up correctly

### ISSUE THREAD

**Issue by Liebeg** [Cuperino/QPrompt-Teleprompter#258]
**Are you ready to report?**

Before reporting a bug, use GitHub's [issues filter](https://github.com/Cuperino/QPrompt/issues) to see if the bug you've found was already reported. If it has been reported, please indicate said bug also affects you and provide details that add value to the discussion. If the bug hasn't been reported, please continue to fill out this form.

**Bug Description**
After converting a .docx to .html inside word in loading it in qprompt it has problems with loading multiple chars like ' for example

**Steps to Reproduce**
1. Convert a docx to html inside word wich contains ' and -
2.load it in qprompt

**Expected behavior**
The characters should show up

**Detailed description of display configuration** (if applicable)

**Screenshots** (optional)
![image](https://github.com/Cuperino/QPrompt-Teleprompter/assets/95579941/3c9ab308-829f-4abc-9640-7588feea6744)

**Device information**
 - QPrompt Version:
![image](https://github.com/Cuperino/QPrompt-Teleprompter/assets/95579941/6937e5f1-8d17-4abb-b5d9-72c9fb534a21)

 - Operating System: win10

**Additional context**

**Comment by Cuperino** [comment]
This looks like a bug that has to do with text encoding. It might be specific to the language of the source document, specific to Windows or both.

- What language is Windows displaying?
- What language does LibreOffice display when you open LibreOffice Writer?
- Could you please upload a document with characters that would get imported incorrectly?

**Comment by videosmith** [comment]
I've seen this from several word processing files (pdf the worst).
You could try copy/paste without formatting to see if that works.

**Comment by Cuperino** [comment]
@videosmith Have you had this issue on Linux?

@Liebeg Could you prepare a list of all the glyphs that have caused this issue for you? (If Videosmith's hypothesis is correct then, there could be endless glyphs with this problem and covering them all is impossible. So it's really a matter of supplying all the glyphs used in the languages users prompt in.

**Comment by videosmith** [comment]
Yes, seems to be an issue with the source document. Although I get most in Word format, many are exports from whatever app the client uses... Powerpoint, Keynote, Google meetings, PDF files... difficult to pinpoint since I'm not given the source documents. I will attempt to locate an old one and upload.

**Comment by Cuperino** [comment]
@liebeg Can you try replicate this issue with the Qt 6 build?

**Comment by Cuperino** [comment]
Hey @Liebeg, have you been able to replicate this issue using one of our Qt 6 builds?

**Comment by Cuperino** [comment]
Info not provided. Closing issue as unable to replicate. Feel free to re-open if issue still occurs for you.
